packages/devtools_app/lib/src/screens/vm_developer/vm_library_display.dart

```diff
@@ -61,46 +61,53 @@ class LibraryDependencies extends StatelessWidget {
   final List<LibraryDependency> dependencies;
 
   List<Row> dependencyRows(BuildContext context) {
-    final rows = <Row>[];
-
-    for (final dep in dependencies) {
-      final row = Row(
-        children: [
-          Flexible(
-            child: SelectableText(
-              _dependencyDescription(dep),
-              style: Theme.of(context).fixedFontStyle,
+    final textStyle = Theme.of(context).fixedFontStyle;
+
+    return <Row>[
+      for (final dep in dependencies)
+        Row(
+          children: [
+            Flexible(
+              child: SelectableText(
+                dependencyDescription(dep),
+                style: textStyle,
+              ),
             ),
-          ),
-        ],
-      );
-
-      rows.add(row);
-    }
-
-    return rows;
+          ],
+        )
+    ];
   }
 
-  String _dependencyDescription(LibraryDependency dependency) {
+  @visibleForTesting
+  String dependencyDescription(LibraryDependency dependency) {
     final description = StringBuffer();
+    final void Function() addSpace =
+        () => description.write(description.isEmpty ? '' : ' ');
 
     if (dependency.isImport != null) {
-      description.write(dependency.isImport! ? 'import ' : 'export ');
+      description.write(dependency.isImport! ? 'import' : 'export');
     }
 
+    addSpace();
+
     description.write(
       dependency.target?.name ?? dependency.target?.uri ?? '<Library name>',
     );
 
-    if (dependency.prefix != null) {
-      final prefix = dependency.prefix!;
+    addSpace();
+
+    final prefix = dependency.prefix;
+
+    if (prefix != null) {
```

Comment:
Nit: collapse this into a single conditional statement.

```dart
if (prefix != null && prefix.isNotEmpty) {
...
}
```

Reply:
Done